Sources/Responses/BytesRepresentable.swift

```diff
@@ -1,3 +1,18 @@
+import Foundation
+
+// public extension Equatable where Self: BytesRepresentable {
+//   public static func == (lhs: Self, rhs: Self) -> Bool {
+//     return lhs.toBytes == rhs.toBytes
+//   }
+// }
+
 public protocol BytesRepresentable {
```

Comment:
You should be able to have protocols inherit from other protocols. I think that's the only way you'll be able to do the Equatable.